Program.cs

```diff
@@ -44,24 +44,41 @@ static void Main(string[] args)
             outputDirectory = string.IsNullOrEmpty(outputDirectory)
                 ? Environment.CurrentDirectory
                 : outputDirectory;
+
+            int currentMetadataBlockEpoch = 0;
+            int currentMetadataTxEpoch = 0;
+            int previousMetadataBlockEpoch = 0;
+            int previousMetadataTxEpoch = 0;
+            currentMetadataBlockEpoch = GetMetaDataEpoch(outputDirectory, "BlockEpoch");
+            currentMetadataTxEpoch = GetMetaDataEpoch(outputDirectory, "TxEpoch");
+            previousMetadataBlockEpoch = GetMetaDataEpoch(outputDirectory, "PreviousBlockEpoch");
+            previousMetadataTxEpoch = GetMetaDataEpoch(outputDirectory, "PreviousTxEpoch");
```

Comment:
The `GetMetaDataEpoch` static method seems to return `0` if any exception occurred (maybe if metadata json file doesn't exist). Then how about declaring variables with assigning `GetMetaDataEpoch` method's return value? Like...

```suggestion
            int currentMetadataBlockEpoch = GetMetaDataEpoch(outputDirectory, "BlockEpoch");
            int currentMetadataTxEpoch = GetMetaDataEpoch(outputDirectory, "TxEpoch");
            int previousMetadataBlockEpoch = GetMetaDataEpoch(outputDirectory, "PreviousBlockEpoch");
            int previousMetadataTxEpoch = GetMetaDataEpoch(outputDirectory, "PreviousTxEpoch");
```